libhb/decavcodec.c

```diff
@@ -2553,6 +2618,21 @@ static int decavcodecvInfo( hb_work_object_t *w, hb_work_info_t *info )
         }
     }
 #endif
+#if HB_PROJECT_FEATURE_VCE
```

Comment:
The other hardware encoders check if the hwaccel falled back to software decoder, wouldn't that work for amfdec too?